README.md

```diff
@@ -6,7 +6,56 @@
 
 Encoding and decoding support for BSON in Rust
 
+## Useful links
+- [API Documentation](https://docs.rs/bson/)
+- [Serde](https://serde.rs/)
+
+## Installation
+This crate works with Cargo and can be found on
+[crates.io](https://crates.io/crates/bson) with a `Cargo.toml` like:
+
 ```toml
 [dependencies]
 bson = "*"
```

Comment:
let's change this while we're here to specify the lib version.

`bson = "0.3.2"`